cylc/flow/network/schema.py

```diff
@@ -1263,7 +1263,39 @@ class NamespaceName(String):
 class NamespaceIDGlob(String):
     """A glob search for an active task or family.
 
-    Can use the wildcard character (`*`), e.g `foo*` might match `foot`.
+    Examples:
+
+    * `foo.*` the task named `foo` in any cycle.
+    * `*.1` all tasks in cycle `1`.
+    * `*.*:failed` all failed tasks.
+
+    Can use either of these formats:
+
+    * `namespace[.cycle][:status]`
+    * `point[/namespace][:status]`
+
+    Where both `namespace` and `cycle` can be globs.
+
+    """
+
+
+class NamespaceJobGlob(String):
```

Comment:
This input type enables us to tell that this mutation works on jobs by flicking through the results of an introspection query.